canon/src/store/host.rs

```diff
@@ -0,0 +1,84 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use blake2b_simd::Params;
+use lazy_static::lazy_static;
+use parking_lot::RwLock;
+
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use crate::{Canon, CanonError, Id, Sink, Source};
+
+struct InMemoryMap(HashMap<Id, Vec<u8>>);
+
+impl InMemoryMap {
+    fn new() -> Self {
+        InMemoryMap(HashMap::new())
+    }
+
+    fn insert(&mut self, id: Id, bytes: Vec<u8>) {
+        self.0.insert(id, bytes);
+    }
+
+    fn get(&self, id: &Id) -> Option<&[u8]> {
+        self.0.get(id).map(AsRef::as_ref)
+    }
+}
+
+lazy_static! {
+    static ref STATIC_MAP: Arc<RwLock<InMemoryMap>> =
+        Arc::new(RwLock::new(InMemoryMap::new()));
+}
+
+pub(crate) struct HostStore;
+
+impl HostStore {
+    pub(crate) fn fetch(_id: &Id, _into: &mut [u8]) -> Result<(), CanonError> {
+        todo!("a");
```

Comment:
If you're going to leave the code as is, add a more meaningful `todo` and maybe a comment.